sdk/storage/src/common/storage_uri_builder.cpp

```diff
@@ -5,6 +5,7 @@
 
 #include <algorithm>
 #include <cctype>
+#include <iterator>
```

Comment:
Saw this in my local dev.
Wasn't sure about it was only my windows dev machine or not 

We should catch this on CI